react/features/conference/components/Conference.web.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react';
+
+/**
+ * For legacy reasons, inline style for display none.
+ * @type {{display: string}}
+ */
+const displayNoneStyle = {
+    display: 'none'
+};
+
+/**
+ * Implements a React Component which renders initial conference layout
+ */
+export default class Conference extends Component {
+
+   /**
+    * Implements React's {@link Component#render()}.
+    *
+    * @inheritdoc
+    * @returns {ReactElement}
+    */
+    render() {
```

Comment:
In at least 4 of the .web.js files I notice that the indentation of method documentation comments is incorrect/inconsistent. In the future, please make sure that the source code of your PRs including its formatting is consistent with the existing source code.

Please don't modify this PR until further notice because I'm currently trying to merge parts of it!